usr/lib/mate-hud/mate-hud

```diff
@@ -536,6 +536,10 @@ def hud(widget, keystr, user_data):
     logging.debug('_GTK_WINDOW_OBJECT_PATH: %s', gtk_win_object_path)
     logging.debug('_UNITY_OBJECT_PATH: %s', gtk_unity_object_path)
 
+    # Get the menu path separator
+    global PATH_SEPARATOR
```

Comment:
This should be moved higher up so that it appears before the actual usages of it.

Reply:
Can you clarify? I don't see the issue.
Sits in main loop in `main` until the keybinding calls `hud()` where `PATH_SEPARATOR` is set before it calls `try_appmenu_interface()` or `try_gtk_interface()` where it's referenced.

Reply:
I meant it in terms of a human reading the code from the top of the file down.